src/flexbox.rs

```diff
@@ -1800,4 +1803,52 @@ mod tests {
         assert_eq!(constants.container_size, Size::ZERO);
         assert_eq!(constants.inner_container_size, Size::ZERO);
     }
+
+    #[test]
+    fn hidden_layout_should_hide_recursively() {
+        let mut taffy = Taffy::new();
+
+        let grandchild_00 = taffy
+            .new_leaf(FlexboxLayout { display: Flex, size: Size::from_points(50.0, 50.0), ..Default::default() })
+            .unwrap();
+
+        let grandchild_01 = taffy
+            .new_leaf(FlexboxLayout { display: Flex, size: Size::from_points(50.0, 50.0), ..Default::default() })
```

Comment:
I think this text would probably be clearer if we captured this `FlexboxLayout` into a local constant.

Reply:
Good spot, didn't think of that :)